src/amoc.app.src

```diff
@@ -7,6 +7,7 @@
                   kernel,
                   stdlib,
                   lager,
+                  jiffy,
```

Comment:
We also need a `jiffy` for `cowboy-swagger` .
So, the question is to replace `jsx` with `jiffy`? I will try.